test/helpers/fixture_sets/basic.rb

```diff
@@ -0,0 +1,63 @@
+module FixtureSetAssertions
+  class Basic < FixtureSet
```

Comment:
TBH I'm not a big fan of naming it `Basic` in this case, because first time when you look at it you think it's a base class like `ActiveRecord::Base`, but then you see it's only about the basic scenario.

As an alternative I can think about moving basic scenario testing into `test/integration/basic_test.rb` and storing all basic helpers right in that file together with tests. This way you can limit basic assertions only to that file.